ckanext/markdown_view/plugin.py

```diff
@@ -0,0 +1,64 @@
+import ckan.plugins as plugins
+import ckan.plugins.toolkit as tk
+import logging
+
+import requests
+import markdown
+
+
+logger = logging.getLogger(__name__)
+
+
+class MarkdownViewPlugin(plugins.SingletonPlugin):
+    plugins.implements(plugins.IConfigurer, inherit=True)
+    plugins.implements(plugins.IResourceView, inherit=True)
+    plugins.implements(plugins.ITemplateHelpers)
+
+    # IConfigurer
+    def update_config(self, config):
+        tk.add_template_directory(config, 'templates')
+
+    def info(self):
+        return {
+            'name': tk._('Markdown'),
+            'icon': 'file-text-o',
+            'filterable': False,
+            'iframed': False,
+        }
+
+    def can_view(self, data_dict):
+        return data_dict['resource'].get('format') == 'text/markdown'
```

Comment:
Can we accept `markdown` in addition to `text/markdown`? The check should probably be case insensitive as well.